js/filter-tags.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+
+var TAGTEMPLATE = _.template(
+  '<li class="tag">' +
+    '{{ text }}' +
+    '<button class="button tag__remove">' +
+      '<span class="u-visually-hidden">Remove</span>' +
+    '</button>' +
+  '</li>',
+  {interpolate: /\{\{(.+?)\}\}/g}
+);
+
+function Tag($input, filter) {
+  this.$input = $input;
+  this.key = $input.attr('id');
+  this.filter = filter;
+  this.checkOrRadio = $input.attr('type') === 'checkbox' || 'radio';
```

Comment:
This might not be doing what you want it to. If this variable is meant to capture whether the input is a checkbox or radio button (true) or not (false), then we might want something like `['checkbox', 'radio'].indexOf($input.attr('type')) !== -1`.